core/spec.md

```diff
@@ -336,10 +338,8 @@ be one of the following data types:
     alphanumeric characters (`[a-z0-9]`), `-`, `_` or a `.`; be no longer
     than 63 characters; start with an alphanumeric character and be unique
     within the scope of this map
-  - When serialized as an HTTP header, each key MUST appear as a separate
-    HTTP header prefixed with `xRegistry-` and the map's owning attribute
-    name. The header value MUST be the key's "value". See
-    [HTTP Header Values](#http-header-values) for more information
+  - See [Serializaing Resource Documents](#serializing-resource-documents)
+    for more information serializing maps as HTTP headers
```

Comment:
"information serialization" - add "about" or "on" between the two words?

Reply:
done